src/inmanta/server/services/compilerservice.py

```diff
@@ -276,7 +276,7 @@ async def run(self, force_update: Optional[bool] = False) -> Tuple[bool, Optiona
             result = await self._run_compile_stage("Recompiling configuration model", cmd, project_dir, env=env_vars_compile)
             success = result.returncode == 0
             if not success:
```

Comment:
you missed one important word: exporting compile 

when self.request.do_export is false, it can stay debug.

Reply:
Changed it in https://github.com/inmanta/inmanta/pull/2475/commits/1c98e58913d8f09979868d60e98213abdd540a66